buildconfig/CMake/ExternalSipPyQt4.cmake

```diff
@@ -51,21 +55,21 @@ ExternalProject_Add(extern-pyqt4
     --no-deprecated
     --qmake=/usr/bin/qmake-qt4
     --no-qsci-api
-  BUILD_COMMAND make 2> build.log
+  BUILD_COMMAND make -j24 2> build.log
```

Comment:
You should use [ProcessorCount](https://cmake.org/cmake/help/v3.0/module/ProcessorCount.html) here

Reply:
Thanks. I actually left that in by accident and meant to check how to do that with cmake.

Reply:
Just pushed fix to use `ProcessorCount`.